utils/deploys/deployToken.ts

```diff
@@ -7,6 +7,8 @@ import { IndexToken__factory } from "../../typechain/factories/IndexToken__facto
 import { MerkleDistributor__factory } from "../../typechain/factories/MerkleDistributor__factory";
 import { Vesting__factory } from "../../typechain/factories/Vesting__factory";
 import { OtcEscrow__factory } from "../../typechain/factories/OtcEscrow__factory";
+import { FTCVesting__factory } from "@typechain/factories/FTCVesting__factory";
```

Comment:
Can you use the `../../typechain` formatting here  so that it falls in line with how the rest of the repo formats it (I agree that your way looks better though)